src/context/playlist-context.js

```diff
@@ -6,7 +6,11 @@ const PlaylistContext=createContext()
 
 const PlaylistProvider=({children})=>{
     const [playlist,setPlaylist]=useState([])
-    return (<PlaylistContext.Provider value={{playlist,setPlaylist}}>{children}</PlaylistContext.Provider>)
+    const [playlistName,setPlaylistName]=useState(false)
+    const [playlistInput,setPlaylistInput]=useState()
+    const [modal,setModal]=useState(false)
+    const [sentPlayList,setSentPlayList]=useState(null)
```

Comment:
since states are more than 3 you can go for reducer

Reply:
will change it in final version.